e2e/portalicious/tests/DoPayment/DoSuccessfulPayment.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test } from '@playwright/test';
+import { format } from 'date-fns';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import NLRCProgram from '@121-service/src/seed-data/program/program-nlrc-ocw.json';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import {
+  programIdOCW,
+  registrationsOCW,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import PaymentsPage from '@121-e2e/portalicious/pages/PaymentsPage';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const accessToken = await getAccessToken();
+  await seedIncludedRegistrations(registrationsOCW, programIdOCW, accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[31970] Do successful payment', async ({ page }) => {
+  const paymentsPage = new PaymentsPage(page);
+  const projectTitle = NLRCProgram.titlePortal.en;
+  const numberOfPas = registrationsOCW.length;
+  const defaultTransferValue = NLRCProgram.fixedTransferValue;
+  const defaultMaxTransferValue = registrationsOCW.reduce((output, pa) => {
+    return output + pa.paymentAmountMultiplier * defaultTransferValue;
+  }, 0);
+  const lastPaymentDate = `${format(new Date(), 'dd/MM/yyyy')}`;
+
+  await test.step('Navigate to Program payments', async () => {
+    await paymentsPage.selectProgram(projectTitle);
+
+    await paymentsPage.navigateToProgramPage('Payments');
+  });
+
+  await test.step('Do payment', async () => {
+    await paymentsPage.createPayment();
+    await paymentsPage.startPayment();
+    // Assert redirection to payment overview page
+    await page.waitForURL((url) =>
+      url.pathname.startsWith('/en-GB/project/3/payments/1'),
+    );
+    // Assert payment overview page by payment date/ title
+    await paymentsPage.validatePaymentsDetailsPageByDate(lastPaymentDate);
+  });
+
+  await test.step('Validate payment card', async () => {
+    await paymentsPage.validateToastMessage('Payment created.');
+    await paymentsPage.navigateToProgramPage('Payments');
+    await paymentsPage.waitForPaymentToComplete();
+    // First try to validate the payment card where system still waits for the response from the PA with Voucher payment method.
+    await paymentsPage.validatePaymentCard({
+      date: lastPaymentDate,
+      paymentAmount: defaultMaxTransferValue,
+      registrationsNumber: numberOfPas,
```

Comment:
I would've expected this to be `numberOfPas - 1`, no?

Reply:
It disappeared after the rebase and now it is the amount. I am not sure now if this was intended.